assets/src/edit-story/app/media/local/test/useContextValueProvider.js

```diff
@@ -52,6 +52,8 @@ const MEDIA_LIST_FROM_GET_MEDIA = [
     title: 'IMAGE',
     alt: 'IMAGE',
     local: false,
+    isPlaceholder: undefined,
+    isTranscoded: false,
```

Comment:
Let's rename this to `isOptimized` everywhere

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/7130/commits/9214c315a458c9484a20fcfa949ca8d0998af9fb